CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 For the v16.1 release series, Cumulus Core will be tested against PostgreSQL v13. Users
 should migrate their datastores to Aurora PostgreSQL 13.12+ compatible data
 stores as soon as possible after upgrading to this release.
+- **CUMULUS-3547**
+  - Updated ECS Cluster `/dev/xvdcz` EBS volumes so they're encrypted.
```

Comment:
What's this entry? Typically backports only have an entry under their version header.

Reply:
@kwartrosa reminder to respond to comments with a commit or explanation. If the author 'resolves' the comments, the reviewer must dig to figure out if they've been addressed.

Reply:
@npauzenga additional entry deleted, since backport ticket only needs an entry in the version header.